### PR TITLE
[CS-2475]: Initial fixes for Android build

### PR DESCRIPTION
**Pull request by danibonilha**
### Description

Makes android at least buildable and without crashes, it's still slow and with multiple UI/UX issues, but at least it's alive

### Screenshots

Screenshot 

Screenshot

### CODE REVIEW

File: src/screens/QRScannerScreen.js [hunk 1]
```diff
@@ -87,47 +82,41 @@ const QRScannerScreen = () => {
         </CameraDimmer>
 
         <Container bottom={0} position="absolute" width="100%">
-          {discoverSheetAvailable ? (
-            android ? (
-              <DiscoverSheet />
-            ) : null
```
Comment (topwebtek7) [inline src/screens/QRScannerScreen.js:93]: @danibonilha just for more context, looks like rainbow used DiscoverSheet for android.. does Sheet handles for android as well? 
anyway will give a try android build on my side shortly

Comment (eshinkawa) [reply src/screens/QRScannerScreen.js:93]: trying to run on my device as well

Comment (danibonilha) [reply src/screens/QRScannerScreen.js:93]: DiscoverSheet is a feature that shows assets in the QR code page, searchBar etc, the one on iOS had already been removed, it's an experimental  feature which was never used, this is the sheet from rainbow app:

image

Comment (eshinkawa) [reply src/screens/QRScannerScreen.js:93]: @danibonilha was able to run after adding this to `build.gradle`
https://stackoverflow.com/a/69050529/2716525

**Comment by eshinkawa** [comment]
@danibonilha was only able to run after adding this to `build.gradle`
https://stackoverflow.com/a/69050529/2716525

**Review by eshinkawa — APPROVED** [review]

**Comment by danibonilha** [comment]
> https://stackoverflow.com/a/69050529/2716525

I did not run into this issue, did you run `yarn setup` before trying to build ? Or it could a m1 thing 🤔 , I'd recommend to clean `node_modules`, and all build folders, then `yarn setup && yarn android`

File: src/screens/QRScannerScreen.js (hunk 1 above)
Comment (topwebtek7) [reply src/screens/QRScannerScreen.js:93]: good job. I was able to run android

**Pull request merged**